### DISCUSSION TITLE
【遇到问题】打包体积优化

### DISCUSSION THREAD

**Discussion by subframe7536** [SPlayer-Dev/SPlayer#133] [想法 Ideas]
### 是网页端还是客户端

客户端

### 当前系统环境

win11

### 当前 Node.js 及 npm 版本

v18.16.0 /v9.6.7

### 当前版本

v2.0.1

### 具体信息

![image](https://github.com/imsyy/SPlayer/assets/78338239/1e11283a-efbb-46c4-8b49-7ba21d54f260)

`app.asar` 解压后发现打包了很多多余的依赖

建议：
1. 只将 `native` 模块放入 `dependencies` ，其余的都放入 `devDependencies` 中
2. 修改 `electron-builder.yml` ，只将 `out` 放入 `files`

**Comment by subframe7536** [comment]
其实更推荐 https://github.com/electron-vite/vite-plugin-electron

**Comment by imsyy** [comment]
主要还是不太熟悉 `electron` 😂
